src/features/deposit/components/DepositForm/index.tsx

```diff
@@ -678,17 +682,19 @@ function renderBalance(
   balance: bigint,
   nativeBalance: bigint
 ) {
-  if (isBaseToken(token)) {
-    if (token.address === "wrap.near") {
-      return balance + nativeBalance
-    }
-    if (token.address === "native") {
-      return nativeBalance
-    }
-    return balance
+  // If the token is base and is wNEAR, return the combined balance of NEAR and wNEAR
```

Comment:
Code should not explain what it does in most cases, because it just duplicates the code. Code may change and the comment most likely won't be changed, which will introduce confusion for a reader. It's better to explain why it does so.